docs/CONTRIBUTING.md

```diff
@@ -361,6 +370,12 @@ Set up the git repo for the crate index by running:
 ./script/init-local-index.sh
 ```
 
+You can then import the database with
+
+```console
+./script/import-database-dump.sh
+```
+
```

Comment:
While it's probably a good idea to document this option, I'm not sure the "Setting up the git index" section is the right place for it 😅